widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -163,6 +165,22 @@ private static void setAuthorization(
         }
     }
 
+    private static void listenCallVisualizerEvents() {
+        Dependencies.glia().getCallVisualizer().on(Omnibrowse.Events.ENGAGEMENT_REQUEST, engagementRequest -> {
+            Consumer<GliaException> onResult = error -> {
+                if (error != null) {
+                    Log.e(TAG, "Error during accepting engagement request, reason" + error.getMessage());
+                }
+            };
+            engagementRequest.accept((String) null, onResult);
+            Log.d(TAG, "Incoming Call Visualizer engagement auto accepted");
```

Comment:
NIT: I think it would be more correct to move this message to the `onResult` callback

Reply:
Fixed 2fe4619f2de6e8e72589ce96d2fa876595989249